libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -180,6 +180,26 @@ describe('CalendarComponent', () => {
     expect(captured.event).toBeUndefined();
   });
 
+  it('should not emit a dateChange event if clicking a date that was not passed in enabledDates', () => {
+    spectator.setInput('enabledDates', ['1997-08-25', '1997-08-26'].map(localMidnightDate));
+    spectator.setInput('selectedDate', localMidnightDate('1997-08-29'));
+
+    const captured = captureDateChangeEvents();
+
+    clickDayOfMonth(27); // August 27st was implicitly disabled
+    expect(captured.event).toBeUndefined();
+  });
```

Comment:
Try to organize the tests according to the AAA pattern - see 'The Good Test' on Kirby Wiki